core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -56,6 +56,14 @@
     margin-bottom: 0;
   }
 
+  .oppia-sharing-links a:focus .share-option-img,
+  .oppia-sharing-links a:focus .embed-link,
+  .oppia-sharing-links-small a:focus .share-option-img,
+  .oppia-sharing-links-small a:focus .embed-link {
+    outline: auto 2px Highlight;
```

Comment:
Highlight is apparently a "deprecated system color"... : https://developer.mozilla.org/en-US/docs/Web/CSS/outline-color

Ditto elsewhere.